### ISSUE TITLE
Decide on which build management tool

### ISSUE THREAD

**Issue by bitionaire** [bitionaire/el-bombillo#5]
Hola, obviously we'll need something very flexible. I would prefer [Gradle](https://gradle.org/) as I already tried to use it in a way where anything could be developed. You can also checkout the article [here](http://www.qudosoft.de/compilecoffeescriptsassandjadewithgradle/) to see how it is used to also execute ruby, Sass compilers etc.

We can also talk with the guys of [itBldz](https://github.com/1and1/itBldz) to check if there's a way to do the same with their tool. By using their tool we can support their development.

So to sum it up I would define our requirements as...
- Runs on differnent OS
- Very flexible (e.g. can execute other build tools and customize the build lifecycle)
- Yet you only need to install max. 2 tools (e.g. JVM + Gradle / Node + itBldz) and can do anything with it

In other words: It ain't Maven

Opinions?
